path_test.go

```diff
@@ -81,3 +82,21 @@ func TestPathCube(t *testing.T) {
 	)
 	proc.Run(t)
 }
+
+func TestDraw_Framecount(t *testing.T) {
+	const (
+		w = 200
+		h = 200
+	)
+	proc := newTestProc(t, w, h,
+		func(proc *Proc) {
+			proc.Fill(color.RGBA{R: 255, A: 255})
+			proc.NoLoop()
+		},
+		func(proc *Proc) {
+			proc.Text(fmt.Sprintf("%d", proc.FrameCount()), 50, 50)
+		},
+		"testdata/framecount.png",
```

Comment:
with the code as is, `framecount_golden.png` displays `1`, as does the p5js` code.
but according to the documentation of `FrameCount` (that returns the number of frames *displayed* since the program's start), some people could interpret the correct answer to be `0`.

some (re)wording of the documentation could be devised (but could also be deferred to another PR) to address this 1-off effect.
I suppose it probably isn't too much a big deal however.